src/dataset-builder/custom-inputs/CreateDatasetInputs.ts

```diff
@@ -0,0 +1,47 @@
+import { useUniqueId } from '@terra-ui-packages/components';
```

Comment:
This is copied over from `data-catalog/create-dataset/CreateDatasetInputs`. Eventually we may want to move to using standard input components, but I wanted to keep this PR strictly to removing catalog code.